events-processor/models/billable_metrics.go

```diff
@@ -8,15 +8,28 @@ import (
 	"github.com/getlago/lago/events-processor/utils"
 )
 
+type AggregationType int
+
+const (
+	AggregationTypeCount       AggregationType = 0
+	AggregationTypeSum         AggregationType = 1
+	AggregationTypeMax         AggregationType = 2
+	AggregationTypeUniqueCount AggregationType = 3
+	AggregationTypeWeightedSum AggregationType = 5
+	AggregationTypeLatest      AggregationType = 6
+	AggregationTypeCustom      AggregationType = 7
+)
```

Comment:
I guess the way you'd usually handle this idiomatically is:

```suggestion
const (
	AggregationTypeCount = itoa
	AggregationTypeSum
	AggregationTypeMax
	AggregationTypeUniqueCount
	AggregationTypeWeightedSum
	AggregationTypeLatest 
	AggregationTypeCustom
)

func (t AggregationType) String() string {
	aggType := ""
	
	switch t {
	case AggregationTypeCount:
		aggType = "count"
	case AggregationTypeSum:
		aggType = "sum"
	case AggregationTypeMax:
		aggType = "max"
	case AggregationTypeUniqueCount:
		aggType = "unique_count"
	case AggregationTypeWeightedSum:
		aggType = "weighted_sum"
	case AggregationTypeLatest:
		aggType = "latest"
	case AggregationTypeCustom:
		aggType = "custom"
	}

	return aggType
}
```